Jenkinsfile

```diff
@@ -0,0 +1,69 @@
+#!groovy
+
+properties([
+    buildDiscarder(logRotator(daysToKeepStr: '20', numToKeepStr: '30')),
+
+    pipelineTriggers([pollSCM('H/15 * * * *')])
+])
+
+def test_ignition(ARCH, GOVERSION)
+{
+    node("${ARCH} && docker") {
+        def CGO = (ARCH == 'arm64') ? 1 : 0
+        withEnv(["TARGET=${ARCH}", "CGO_ENABLED=${CGO}",
+                 "GOARCH=${ARCH}", "GOVERSION=${GOVERSION}"]) {
+            stage("SCM $GOVERSION") {
+                checkout scm
+            }
+
+            stage("Build & Test $GOVERSION") {
+                sh '''#!/bin/bash -ex
```

Comment:
I added the integration-tests script for the later shell block which takes care of actually running the tests.